generate/scope_generator.go

```diff
@@ -39,7 +39,7 @@ var requiredScopeKeys = [3]string{"description", "issuedBy", "acceptedBy"}
 
 func handleCaseConversion(s string) string {
 	var camelCase string
-	nextCap := false
+	nextCap := true // default as true so we capitalize the first letter
```

Comment:
What happens to the other generators that use this function? Does switching this to false have ramifications for them?

Reply:
So I looked into that when I changed this. The only other reference to this function capitalized the first letter by hand outside of it. This way there is no need for it to do that.

What I did forget to do was remove the (now redundant) line that did the capitalization manually. Making that change now